.github/workflows/maven-deploy.yml

```diff
@@ -30,6 +30,9 @@ jobs:
           server-username: NEXUS_USERNAME
           server-password: NEXUS_PASSWORD
 
+      - name : Check Maven and Java versions
+        run : mvn -v
```

Comment:
In case you didn't already notice the [Lint failure](https://github.com/xspec/xspec/actions/runs/11027440981/job/30625746335?pr=2009): the "prettier" program does not like the spaces before the colons on these two lines.

Reply:
You fixed this in 14b861088aefa6eaaf421311426e0a1b16bf7e28